_posts/2020-01-30-demystifying_samesite_cookie_updates/index.md

```diff
@@ -0,0 +1,129 @@
+
+- If the cookie is being transmitted cross-site and the server does not accept secure connections (HTTPS) then the cookie can no longer be sent.
+- There are some old browsers which are not compatible with `SameSite=None; Secure` cookies, the Chromium team have [published a list](https://www.chromium.org/updates/same-site/incompatible-clients).
+
+Most websites are already operating on secure (HTTPS) connections which means that the `Secure` attribute doesn't pose much worry, but it is worth ensuring communication with any third party servers is being done securely if the site is still accessible on insecure (HTTP) connections. Due to the older, incompatible browsers you may experience data usually contained within cookies from these browsers not working as expected. In the case of Snowplow, this will mean new `network_userid`s will be generated for requests from these incompatible browsers and will skew user counts that are relying solely on the `network_userid`.
+
+One type of request that is likely going to be affected by these two changes are third party tracking provider requests. The providers will often be running on a different domain and may not be including the `SameSite` attribute on cookies. This means any tracking that relies on these cookies has the potential to stop working. _You may need to take action to ensure cross-site tracking contains to work in Chrome 80._
+
+### Why it is happening now
+
+The new default is designed to better protect everyone's privacy online, as well as offering sites that are delivering cookies an avenue to explicitly label them so that they are never sent in any cross-site requests. By changing the `SameSite` attribute to `Lax` by default, the Chromium team are ensuring that third party services must label their cookies in a way that will allow them to be sent cross-site.
+
+There has been a proposal which is refered to as [Incrementally Better Cookies](https://tools.ietf.org/html/draft-west-cookie-incrementalism-00) that was published last year and this is one of the first steps towards that. It is expected that other browsers will also take this, or similar, steps toward handling cookies in this way.
+
+<h3 id="what-it-means">What it means for your tracking</h3>
+
+If you are relying on cookies to identify users then this may stop working for users who browse sites with Chrome (and other Chromium based browsers). If the cookie that has been stored does not contain the `SameSite=None` attribute then the cookie will not be sent in any requests to the third party server.
+
+### How to check if you are affected
+
+There are two types of requests that we generally talk about, first party and third party. A first party request is one that is sent to the same domain that the website is being served from (i.e. the same as the domain visible in the browsers address bar). Whereas a third party request is one which is sent to a domain that is different from the one visible in the browsers address bar.
```

Comment:
I find the use of 'request' to mean traffic going either way confusing when trying to understand 1st vs 3rd party context. It is of course technically correct but to me doesn't help unambiguously identify the two players: the site that is being visited and the collector. But this might be just a personal thing...